agent/engine/task_manager.go

```diff
@@ -254,7 +254,7 @@ func (mtask *managedTask) handleContainerChange(containerChange dockerContainerC
 			}
 			// If docker returned a transient error while trying to stop a container,
 			// reset the known status to the current status and return
-			cannotStopContainerError, ok := event.Error.(*CannotStopContainerError)
+			cannotStopContainerError, ok := event.Error.(CannotStopContainerError)
```

Comment:
Pretty much exactly what I'm doing... :1st_place_medal: 

Except that I'm taking the opportunity to change that method name!